### ISSUE TITLE
Feature Request : Auto correct word insert

### ISSUE THREAD

**Issue by muhelen** [mmmewk/crosswordle#35]
Another feedback I'm adding for your consideration :
_Currently, say we figure out the word in a row or a column correctly, that gets populated. 
If one of the letters in that word is a part of the next word in a column or row, we need to type it over again. Can this be avoided?_

**Comment by mmmewk** [comment]
Yeah I've been thinking about this it would be nice to have. Only problem is that sometimes you want to guess a word that you know is not correct as a way to knock out possible letters.

Especially with some of the puzzles having 3 letter words its important that users to be able to guess a word that is incorrect as a way to get more information

**Comment by muhelen** [comment]
Yeah, may be can allow to overwrite it with another letter, if they wanted to guess a different letter..
